org.ekstep.uploadfile-1.0/editor/uploadapp.js

```diff
@@ -37,14 +37,6 @@ angular.module('org.ekstep.uploadfile-1.0', []).controller('uploadfileController
                         $("#orLabel").show();
                     }
                 },
-                onSubmit: function (id, name, responseJSON) {
-                    $('#qq-upload-actions').hide();
-                    $("#url-upload").hide();
-                    $("#orLabel").hide();
-                    $scope.showLoader(true);
-                    $scope.uploadBtn = false;
-                    $scope.uploadFile($scope.callback);
```

Comment:
@vinukumar-vs We kept the config `autoUpload` to `false` only, but we were overriding the `onSubmit` event, in which we were making API call. So instead of this, I put this API call on click of `Upload` button only.